Social-Media-Guidelines.md

```diff
@@ -0,0 +1,77 @@
+# Social Media Guidelines
+
+## The [@bitcoin_design](https://twitter.com/bitcoin_design) Twitter account
+
+The twitter account gives more exposure to designers and efforts in the community. It also hosts the community call every 3 weeks via Twitter Spaces.
+
+### Format
+
+Voice, tone & writing style:
+
+- FOR things, promoting solutions, using positive language
+- Not AGAINST things, or use negative language
+- Informative & educational
+- High signal-to-noise ratio
+- Friendly & approachable
+- Succinct
+- Use threads with each Tweet capturing one idea/point
+
+Content ideas:
+
+- Guide highlights: When new content is added to the guide, create a new thread on the additions (or re-tweet the authors threads - probably better). 
+- Designer spotlight: highlight a designer in the community and share a short story about them / why they like bitcoin / what they work on etc.
+- Announce calls and call recordings (learning bitcoin & design, design review, community call…)
+- Quick user research and polls for community projects
+- Respond to current design-related conversations (e.g. post a link to the Units & Symbols page if there’s a thread somewhere around unit formatting)
+- Add context to Tweets when RT'ing to add value
+
+Do's:
+
+- OK to highlight when products, services, and brands adopt strategies from the Guide, or do things that’s good for open source bitcoin or open design processes
+
+Don'ts:
+
+- Keep direct criticism to personal accounts
+- Keep direct promotion to personal accounts
+
+Good:
+
+- ABC Wallet just launched a new version with an improved UI. It's a perfect example of techniques from the Onboarding section of the Guide!
+- Just wrapped up another awesome design sprint with XYZ Wallet. Thanks for being so open to collaboration with the Bitcoin Design Community!
+- Several members of the Design Community participated in this podcast about DEF Wallet to talk about this Lightning backups. What do you think about this topic?
+- MNO Exchange just launched an initiative to provide grant funding to people working on open source bitcoin projects. Calling all designers!
+
+Bad:
+
+- GHI Wallet has a pretty bad UI. Shame they haven't read the Guide.
+- Gee, JKL Wallet, would it kill you to join a Design Review call?
+- PQR Wallet is the best. You should all download it now!
+- STU Wallet has a promotion going on where you can get free sats for downloading their app.
+
+### Editors
+
+The Twitter account is currently managed by Bosch.
```

Comment:
```suggestion
The Twitter account is currently managed by [bosch](https://twitter.com/_Bosch_).
```

Added a link to my twitter in case people need to contact me for whatever reason.